### DISCUSSION TITLE
How would one go about scraping and infinite scoll page?

### DISCUSSION THREAD

**Discussion by gevera** [elixir-crawly/crawly#264] [Q&A]
I would like to scrape a website that uses infinite scroll for its feed page. Unfortunately It doesn't use any pagination. Crawly parses the fist 20 items, and then it stops. I wounder if there is a way to trigger a scroll or some event to load the next 20 articles, and so on. Appreciate your help

**Comment by oltarasenko** [comment]
Well, usually you would have to mimic requests the page is doing.

Maybe you can show the website/page so we could try to help you?
